cylc/rose/utilities.py

```diff
@@ -442,3 +442,31 @@ def simplify_opts_strings(opts):
             seen_once.append(item)
 
     return ' '.join(reversed(seen_once))
+
+
+def dump_rose_log(dest_root, node):
+    from metomi.rose.config import ConfigDumper
+    from metomi.isodatetime.datetimeoper import DateTimeOperator
```

Comment:
Have already imported `metomi.rose.config` at the top of the file, fine to move these both up there.